src/main/c/ssl.c

```diff
@@ -1178,11 +1178,11 @@ TCN_IMPLEMENT_CALL(jint, SSL, getLastErrorNumber)(TCN_STDARGS) {
 }
 
 static void ssl_info_callback(const SSL *ssl, int where, int ret) {
-    tcn_ssl_ctxt_t *c;
+    int *handshakeCount;
```

Comment:
Don't need to no.  But if you were to use it with out "really" initializing it you would get a quick stack trace....as opposed to accepting what ever gets put in there and potentially dereferencing who knows what.  This block of code is good as it is initialized before it is used...but more just good practice (declaration is initialization).